### DISCUSSION TITLE
What value is reported for k-Fold Validation?

### DISCUSSION THREAD

**Discussion by filipsPL** [mljar/mljar-supervised#510] [Q&A] [answered]
Hi,

I perform k-fold cross-validation in `mode="Optuna"`. My question is what metric value (eg AUC) is reported in the outputs (`README.md` or `leaderboard.csv`)? I guess this is **the average from all folds**, but this is not stated explicitly anywhere. 

If so, reporting standard deviation would be also useful?

best,
Filip

**Comment by pplonski** [comment, marked as answer]
Hi @filipsPL,

The metrics are computed from all samples in the case of cross-validation. The AutoML gathers predictions from all folds and then computes the metric.

**Reply by filipsPL** [reply]
Thanks for a quick reply.

Just to make sure: the algoritm is as follows (eg for 3-fold CV and AUC):

1. For each fold the predictions are made:
```
fold1 -> predictions1
fold2 -> predictions2
fold3 -> predictions3
```

2. all predicted values (predictions1 + predictions2 + predictions3) are combined 
3. AUC is calculated

?

**Reply by pplonski** [reply]
Yes

**Reply by filipsPL** [reply]
Thanks!
